en/docs/develop/oidc-scope-management-rest-apis.md

```diff
@@ -3,14 +3,20 @@ template: templates/swagger.html
 ---
 # OpenID Connect Scope Management Rest API Definition - v1
 
-??? Note "Note: Click to view"
-    Do the following to try out the REST APIs with your local instance of WSO2 Identity Server.
-
-    1.  Expand the relevant API operation and click the **Try It Out** button.
-    2.  Fill in relevant sample values for the input parameters and click **Execute**.
-        You will receive a sample curl command with the sample values you filled in.
-    3. Add a `-k` header to the curl command and run the curl command on the terminal with a running instance of WSO2 IS.
+The OIDC scope API in WSO2 Identity Server can be used to manage OIDC scopes and scope claim mapping. OIDC scope is a
+sub set of OAuth2 scopes. Refer OAUTH2 Scope endpoint [here](oauth2-scope-management-rest-apis.md). 
+Hence Identity Server won't allow same scope name across these two (OAUTH2 Scopes & OIDC Scopes) endpoints.
```

Comment:
Hence, WSO2 Identity Server will not allow using the same scope name across these two (OAuth2 scope and OIDC scope) endpoints. For more information about the OAuth2 scope endpoint, see `[OAuth2 Scope Management REST APIs](../../develop/oauth2-scope-management-rest-apis)`.